File: ERP_API Ty Thac/ERP_API/Models/LoginResult.cs
namespace ERP_API.Models
{
    public class LoginResult
    {
        public Boolean Result { get; set; }

        public string? Status { get; set; }

        public string? UserID { get; set; }

        public string? UserName { get; set; }

        public string? Group { get; set; }

        public string? Department { get; set; }

        public string? Factory { get; set; }
    }
}

File: TyThac/lib/models/login_result.dart
class LoginResult {
  final String result;
  final String status;
  final String userID;
  final String userName;
  final String group;

  const LoginResult({
    required this.result,
    required this.status,
    required this.userID,
    required this.userName,
    required this.group,
  });

  factory LoginResult.fromJson(json) {
    return LoginResult(
      result: json['result'],
      status: json['status'],
      userID: json['userID'],
      userName: json['userName'],
      group: json['group'],
    );
  }
}